src/java.xml/share/classes/module-info.java

```diff
@@ -27,11 +27,70 @@
  * Defines the Java API for XML Processing (JAXP), the Streaming API for XML (StAX),
  * the Simple API for XML (SAX), and the W3C Document Object Model (DOM) API.
  *
+ * <h2 id="ConfigurationFile">JAXP Configuration File</h2>
+ * JAXP supports the use of a configuration file for the
+ * <a href="#LookupMechanism">Factory Lookup Mechanism</a> and
+ * setting properties that have defined corresponding system properties.
+ *
+ * <h3>Format</h3>
+ * The configuration file must be in standard {@link java.util.Properties} format.
+ * <p>
+ * The keys are the names of the system properties, for example, those listed in
+ * column {@code System Property Name} of the table <a href="#Factories">JAXP Factories</a>,
+ * or {@code System Property} in the table {@code Catalog Features}
+ * of class {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}.
+ *
+ * <p>
+ * The values are those defined in the specific API or process, for example, the
+ * fully qualified name of the implementation class for the
+ * <a href="#LookupMechanism">Factory Lookup Mechanism</a>, or {@code value} in
+ * table {@code Catalog Features} of class
+ * {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}.
+ *
+ * <p>
+ * Below are examples on what can be placed in the configuration file:
+ * <pre>
+ *     {@code javax.xml.parsers.DocumentBuilderFactory=packagename.DocumentBuilderFactoryImpl}
+ *     {@code javax.xml.catalog.resolve=strict}
+ * </pre>
+ *
+ *
+ * <h3 id="CF_Default">{@code jaxp.properties}</h3>
+ * By default, the <a href="#Factories">JAXP Factories</a> will look for a
+ * configuration file called {@code jaxp.properties} in the {@code conf} directory
+ * of the Java installation and use the entries if any to customize the behavior
```

Comment:
"of the Java installation" should probably say the run-time image, or maybe just change this to use ${java.home}/conf as it is used in other API docs.

Reply:
changed to ${java.home}/conf